src/components/AssetHeader/AssetActions.tsx

```diff
@@ -59,7 +59,7 @@ export const AssetActions: React.FC<AssetActionProps> = ({
   cryptoBalance,
   isMobile,
 }) => {
-  const history = useHistory()
+  const navigate = useNavigate()
```

Comment:
`history.push()` and similar methods now gone and `navigate()` now handles everything (push, going back and forward)